test/browser/features/DVLADrivingLicence.feature

```diff
@@ -278,6 +298,104 @@ Feature: DVLA Driving licence CRI Error Validations
+    And I check the page Title Error: Enter your details exactly as they appear on your UK driving licence – Prove your identity – GOV.UK
     Examples:
       |DrivingLicenceSubject          |InvalidValidToDay|InvalidValidToMonth|InvalidValidToYear|
       |DrivingLicenceSubjectHappyPeter|         10      |     01            |         2010     |
+
+  @mock-api:dl-success @validation-regression @build @staging
+  Scenario Outline:  DVLA Driving Licence error validation when DVLA consent checkbox is unselected
+    Given User enters DVLA data as a <DrivingLicenceSubject>
+    And DVLA consent checkbox is unselected
+    When User clicks on continue
+    And I can see the DVLA consent error on the checkbox as Error: You must give your consent to continue
+    Then I can see the DVLA consent error summary as You must give your consent to continue
+    Examples:
+      |DrivingLicenceSubject             |
+      |DrivingLicenceSubjectHappyPeter   |
+
+  @mock-api:dl-failed @validation-regression @build @staging
+  Scenario Outline: DVLA Driving Licence number validation test - Correct licence number structure - error validation
+    Given User enters DVLA data as a <DrivingLicenceSubject>
+    And User re-enters drivingLicenceNumber as <InvalidLicenceNumber>
+    When User clicks on continue
+    Then Proper error message is displayed as We could not find your details
+    Examples:
+      |DrivingLicenceSubject          | InvalidLicenceNumber  |
+      |DrivingLicenceSubjectHappyKenneth| DECER657085K99LN      |
+
+  @mock-api:dl-failed @validation-regression @build @staging
+  Scenario Outline:  DVLA Driving Licence number validation test - (VALID, female licenceNumber DOB Jan)
+    Given User enters DVLA data as a <DrivingLicenceSubject>
+    And User re-enters drivingLicenceNumber as <InvalidLicenceNumber>
+    And User re-enters day of birth as <InvalidDayOfBirth>
+    And User re-enters month of birth as <InvalidMonthOfBirth>
+    And User re-enters year of birth as <InvalidYearOfBirth>
+    When User clicks on continue
+    Then Proper error message is displayed as We could not find your details
+    Examples:
+      |DrivingLicenceSubject            |InvalidLicenceNumber|InvalidDayOfBirth|InvalidMonthOfBirth|InvalidYearOfBirth|
+      |DrivingLicenceSubjectHappyKenneth| DECER651085K99LN |         08      |        01       |        1965      |
+
+  @mock-api:dl-failed @validation-regression @build @staging
+  Scenario Outline:  DVLA Driving Licence number validation test - (VALID, female licenceNumber DOB Dec)
+    Given User enters DVLA data as a <DrivingLicenceSubject>
+    And User re-enters drivingLicenceNumber as <InvalidLicenceNumber>
+    And User re-enters day of birth as <InvalidDayOfBirth>
+    And User re-enters month of birth as <InvalidMonthOfBirth>
+    And User re-enters year of birth as <InvalidYearOfBirth>
+    When User clicks on continue
+    Then Proper error message is displayed as We could not find your details
```

Comment:
Is this correct should this not trigger the DoB and doc number validation message?

Reply:
Ignore this my validation logic was wrong